doc/api/perf_hooks.md

```diff
@@ -43,6 +43,14 @@ added: v8.5.0
 An object that can be used to collect performance metrics from the current
 Node.js instance. It is similar to [`window.performance`][] in browsers.
 
+### `performance.toJSON()`
```

Comment:
should be after `performance.timerify` (alphabetical order)

Reply:
Cool making the change. One more question shouldn't be thrown by linter instead of a PR review comment 😅 (Just a feature suggestion which can be implemented).